README.md

```diff
@@ -30,11 +30,11 @@ Adds a `pp(Any?)` and `T.pp()` method to pretty print any Java or Kotlin object.
 
 `pp(obj: Any?, tabSize: Int = 2, printStream: PrintStream = System.out, wrappedLineWidth: Int = 80)`
 
-`tabSize` allows you to change the number of spaces used to indent each level of the output. Default is `2`.
+`tabSize` changes the number of spaces used to indent each level of the output. Default is `2`.
 
-`printStream` allows you to change the `PrintStream` you are printing to. Default is `System.out`.
+`printStream` changes the `PrintStream` being used. Default is `System.out`.
```

Comment:
I'd rather keep `printing to` or something similar. `Being used` is kind of ambiguous.